src/client/mixins/network.ts

```diff
@@ -68,7 +68,7 @@ export class NetworkAware {
 
         for (const [key, value] of Object.entries(actualParams)) {
           if (expectedParams.hasOwnProperty(key) && matched) {
-            matched = expectedParams[key] == value;
+            matched = expectedParams[key] == String(value);
```

Comment:
- [ ] I suspect we'll want to apply similar logic to what is contained in the util package here, since `true == 'true'` is...  `false` 🤦‍♂ 

We could either just `String()` both values or actually include the util package and run a `compare()` against the two values with a `be` operator.